Makefile

```diff
@@ -7,10 +7,10 @@ build:
 redeploy-container:
 	docker-compose build --pull
 	docker-compose run --rm web yarn install
-	docker-compose run --rm web bash -c "bundle exec rake db:migrate && bundle exec rake assets:precompile && bundle exec rake tmp:cache:clear"
-	docker-compose down --remove-orphans
-	rm -f ./tmp/pids/server.pid
+	# docker-compose down --remove-orphans
```

Comment:
I'm not 100% sure this part of the change is necessary. The idea was to let this run *without* affecting the running container so that any downtime is minimized.

Reply:
I removed it because the `rm` command is already executed in the `start.sh` file and the down cmd removal was a mistake :) (I removed it to debug). I switch run to exec because run cmd creates a new container instead of executing the desired commands in the plots2 container.

Reply:
Quoting the discussion in Gitter:

"
Using docker-compose run forces the creation of a new container, thus running rake db:migrate && bundle exec rake assets:precompile ....... will create a new web container and run those commands there, instead of running those commands in the actual plots2 web container
"